keen/saved_queries.py

```diff
@@ -1,5 +1,11 @@
 
+try:
```

Comment:
Bummer, it seems like Mapping was moved to `collections.abc` in version 3.3, so our Travis build complained in 3.2, which is nice: https://docs.python.org/3/library/collections.abc.html

So I'll switch back to [EAFP](https://docs.python.org/3/glossary.html#term-eafp) rather than [LBYL](https://docs.python.org/3/glossary.html#term-lbyl). Pythoners will be happy to see more try/except blocks :)